### PR TITLE
Release appcues template app test

### PR DESCRIPTION
**Pull request by orangejenny**
General availability release of https://github.com/dimagi/commcare-hq/pull/21225

code buddy @pr33thi 
or @millerdev / @jmtroth0 since you two reviewed the last PR

Hold off on merge until I do a bit more testing on staging.

### CODE REVIEW

**Review by pr33thi — APPROVED** [review]

File: corehq/apps/accounting/utils.py [hunk 1]
```diff
@@ -143,7 +143,7 @@ def domain_has_privilege(domain, privilege_slug, **assignment):
 def domain_is_on_trial(domain_name):
```
Comment (jmtroth0) [inline corehq/apps/accounting/utils.py]: This function sounds like an exciting legal thriller

**Review by jmtroth0 — APPROVED** [review]
🎈

**Review by millerdev — APPROVED** [review]

File: corehq/apps/app_manager/views/apps.py [hunk 2]
```diff
@@ -397,24 +397,36 @@ def _inner(request, link_domain, data, master_domain=domain):
...
-            media_class = CommCareMultimedia.get_class_by_data(data)
-            if media_class:
-                multimedia = media_class.get_by_data(data)
-                multimedia.attach_data(data,
-                                       original_filename=os.path.basename(path),
-                                       username=request.user.username)
-                multimedia.add_domain(domain, owner=True)
-                app.create_mapping(multimedia, MULTIMEDIA_PREFIX + path)
+    # Fetch multimedia, which is hosted elsewhere
+    multimedia_filename = os.path.join(app_template_dir(slug), 'multimedia.json')
```
[the remark this answers is not in the document]
Comment (orangejenny) [reply corehq/apps/app_manager/views/apps.py]: yep, that's exactly what's going on

**Comment by orangejenny** [comment]
Added https://github.com/dimagi/commcare-hq/pull/21849/commits/4fb6a70bd12eb775d28aafc7b8905e9ab8e78178 to limit to saas users.

**Pull request merged**